### DISCUSSION TITLE
Why the Charecter 's error_symbol property not showing ?

### DISCUSSION THREAD

**Discussion by sourikghosh** [starship/starship#2086] [Q&A] [answered]
### My ENV

- ### Starship.toml
 `[character] 
success_symbol="[🚀](bold green) "
error_symbol="[✗](bold red) "`

- ### Running bash on Wsl2
- ### windows terminal v1.4.3243.0 and Cascadia Code PL

_It shows the success_symbol even after an error_ 
I used `$?` to check the error code which is greater than 0

### Error_symbol does not show
![Screenshot (441)](https://user-images.githubusercontent.com/61813998/103756359-44a3b400-5035-11eb-9a57-2aa54e7a961b.png)

### Just to prove that Cascadia Code PL supports that Glyphs
![Screenshot (442)](https://user-images.githubusercontent.com/61813998/103756288-2938a900-5035-11eb-8c52-76115e3c9779.png)

**Comment by chipbuster** [comment, marked as answer]
Regression caused by #1897. Could you please open a bug report issue so that we have an easier time tracking this?
